docs/contributing/create-a-starter.md

```diff
@@ -1,9 +1,74 @@
-issue: https://github.com/gatsbyjs/gatsby/issues/8103
 ---
 
-This is a stub. Help our community expand it.
+[Starters](/docs/starters/) are boilerplate projects that Gatsby developers can use to set up a new site quickly. Before creating a starter, it may be helpful to peruse the [Gatsby Starter Library](/starters/) to see what already exists and determine how your starter will provide value.
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Basic requirements
+
+For a starter to work properly, it needs to include some files (see the [Hello World starter](https://github.com/gatsbyjs/gatsby-starter-hello-world/) for a barebones example):
+
+- `README.md`: instructions for how to install and configure your starter, a list of its features or structure, and any helpful tips.
+- `package.json`: the "command center" for Gatsby dependencies and scripts. Find an example in the [Hello World Starter](https://github.com/gatsbyjs/gatsby-starter-hello-world/blob/master/package.json).
+- `gatsby-config.js`: a space to add configurable data and plugins. See [Gatsby Config](/docs/gatsby-config/) for more information.
+- `src/pages`: a directory for page components to live, with at least [one example](https://github.com/gatsbyjs/gatsby-starter-hello-world/blob/master/src/pages/index.js).
+- `static`: a directory for static assets, such as a `favicon.ico` file.
+- `.gitignore` _(optional)_: a file telling Git which resources to leave out of source control, such as the `node_modules` directory, log files, Gatsby `.cache` and `public` directories.
+- `.prettierrc` _(optional)_: a configuration file for [Prettier](https://prettier.io/), a JavaScript linter and formatter used for Gatsby development.
+
+Your starter should also have these qualities:
+
+- Open source and available from a stable URL
+- Configurable
+- Fast
+- Web accessible
+
+Let's expand upon these items to prepare you for creating a winning starter.
+
+## Open source and available from a stable URL
+
+The Gatsby CLI allows users to install a new site with a starter using the command `gatsby new <starter-url>`. For this to work, your starter needs to be available to download. The easiest way to accomplish this is to host your starter on GitHub or Gitlab and use the publicly available repo URL, such as:
+
+`gatsby new https://github.com/gatsbyjs/gatsby-starter-blog`
+
+Although the official starters live in the Gatsby repo, community members can offer their own starters from their own repos. Here's an example of installing a community starter:
+
+`gatsby new https://github.com/netlify-templates/gatsby-starter-netlify-cms`
+
+## Configurable
+
+Starters should utilize metadata in `gatsby-config.js` wherever possible, as this is typically the first place users will look for site configuration information. Some examples of things you could make configurable in `gatsby-config` include:
+
+- Site title
+- Basic color palette
```

Comment:
You normally don't put such information into the gatsby-config but rather into it's own file. You could change that aspect to a new paragraph:

When using things like styled-components it's highly appreciated if the built-in theming capabilities are used and a "theme" file is exposed for configuration.

Reply:
Sounds good, thanks @LekoArts!